live-examples/js-examples/date/date-settime.js

```diff
@@ -1,10 +1,13 @@
-const event1 = new Date('July 1, 1999');
-const event2 = new Date();
-event2.setTime(event1.getTime());
+const launchDate = new Date('July 1, 1999, 12:00:00');
+const dateInSpace = new Date();
```

Comment:
Hey @NiedziolkaMichal! Thank you so much for all your wonderful contributions. The variable name `dateInSpace`, do you mean `futureDate` perhaps?

Reply:
By `dateInSpace` I ment few minutes after launch of some spaceship, when it passed Kármán line. Variable name doesn't really matter, so I changed it to `futureDate` as you suggested. Thanks